src/main/webapp/stats.html

```diff
@@ -3,27 +3,7 @@
 <head>
```

Comment:
nit: <head> should be tabbed in since it is inside the open tag <html>

Reply:
This is also the case for the closing tags </dov> </body> and </html>

Reply:
I have updated the pull request! Could you please review it again? Thank you so much!